trade_remedies_caseworker/templates/v2/merge_organisations/confirm_not_duplicate.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'v2/govuk/base.html' %}
+
+{% block page_title %}Confirm not duplicate{% endblock page_title %}
+
+
+{% block row_content %}
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+            Confirm not a duplicate
```

Comment:
Mismatch between title and heading.

```suggestion
{% block page_title %}Confirm not a duplicate{% endblock page_title %}


{% block row_content %}
    <div class="govuk-grid-column-two-thirds">
        <h1 class="govuk-heading-xl">
            Confirm not a duplicate
```